qa-tests-backend/src/test/groovy/DeploymentEventGraphQLTest.groovy

```diff
@@ -125,7 +125,7 @@ class DeploymentEventGraphQLTest extends BaseSpecification {
             log.info "return code " + depEvents.getCode()
             assert depEvents.getValue().result != null
             def events = depEvents.getValue().result
-            assert events.numPolicyViolations == 1
```

Comment:
IIUC this test expects only "Ubuntu Package Manager Execution" to be triggered. But under P/Z more than that is triggered. That works for other cluster flavors, so I would prefer this to be a:
```
if P/Z
  >= 1
else
  == 1
```
Even better would be to explain what other policies are triggered and why this needs to be >= 1 on P/Z.

Reply:
ACK - will make the changes